### PR TITLE
Bugfix/crash on missing users json

### PR DESCRIPTION
**Pull request by OrBin**

### CODE REVIEW

File: gramhopper/users_helper.py [hunk 1]
```diff
@@ -17,8 +17,9 @@ def __init__(self, file_path: Union[PathLike, str, bytes] = None):
 
     def load_users(self, force: bool = False):
         if force or not self._users:
-            with open(self.file_path, 'r') as file:
-                self._users = json.load(file)
+            if os.path.exists(self.file_path):
+                with open(self.file_path, 'r') as file:
+                    self._users = json.load(file)
```
Comment (meirhalachmi) [inline gramhopper/users_helper.py]: What happens if such file doesn't exist? No users will be defined?

File: setup.py [hunk 2]
```diff
@@ -5,7 +5,7 @@
 
 setup(
     name='gramhopper',
-    version='1.0.6',
+    version='1.0.7',
```
Comment (meirhalachmi) [inline setup.py]: Can't the version number be defined once somewhere?

Comment (OrBin) [reply setup.py]: I hope it can. I'll find a more convenient way to handle it later on.

File: gramhopper/users_helper.py (hunk 1 above)
Comment (OrBin) [reply gramhopper/users_helper.py]: After this change? Yes. No users will be defined (the dictionary of users will be empty).

Comment (meirhalachmi) [reply gramhopper/users_helper.py]: K

File: setup.py (hunk 2 above)
Comment (meirhalachmi) [reply setup.py]: KK

**Pull request merged**